src/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -452,7 +452,7 @@ default int scrub(boolean disableSnapshot, boolean skipCorrupted, boolean checkD
      * The entire sstable will be read to ensure each cell validates if extendedVerify is true
      */
     public int verify(boolean extendedVerify, String keyspaceName, String... tableNames) throws IOException, ExecutionException, InterruptedException;
-    public int verify(boolean extendedVerify, boolean checkVersion, boolean diskFailurePolicy, boolean mutateRepairStatus, boolean checkOwnsTokens, boolean quick, String keyspaceName, String... tableNames) throws IOException, ExecutionException, InterruptedException;
+    public int verify(boolean extendedVerify, boolean checkVersion, boolean diskFailurePolicy, boolean mutateRepairStatus, boolean checkOwnsTokens, boolean quick, boolean onlySai, boolean includeSai, String keyspaceName, String... tableNames) throws IOException, ExecutionException, InterruptedException;
```

Comment:
this is a breaking change, won't this impact sidecar?

Reply:
yeah, thats what we do with repair; we create a new one and have the old ones just call the new one with the defaults.